### ISSUE TITLE
Order of operations incorrect

### ISSUE THREAD

**Issue by kennyjwilli** [xapix-io/axel-f#63]
This answer should be `0.3` and it is `0.09999999999999998`.

```clojure
((axel-f/compile "MAX(0.0, (0.6 * (1.0 - 0.5)))"))
=> 0.09999999999999998
```

**Comment by DeLaGuardo** [comment]
Thanks, will fix it asap

**Comment by DeLaGuardo** [comment]
Fixed in 1.0.5
https://github.com/xapix-io/axel-f/blob/master/test/axel_f/issue_63_test.cljc#L6-L8
